java/splitter/src/test/java/sleeper/splitter/SplitPartitionIT.java

```diff
@@ -128,7 +128,7 @@ public void shouldSplitPartitionForIntKeyCorrectly()
             IngestRecordsFromIterator ingestRecordsFromIterator = new IngestRecordsFromIterator(properties,
                     records.iterator()
             );
-            ingestRecordsFromIterator.write();
+            ingestRecordsFromIterator.write().getNumberOfRecords();
```

Comment:
We don't need to call getNumberOfRecords here.

There are a lot of these in this file so I'll just leave this one comment for the file.